source/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/assets/images/favicon/mstile-150x150.png"/>
+            <TileColor>#da532c</TileColor>
+        </tile>
+    </msapplication>
+</browserconfig>
```

Comment:
What is this?

Reply:
This is android os specific, to improve how shortcut on mobile looks like (though i am not sure anyone is saving rector's link on his mobile 😄 ) it was autogenerated by the tool i used to create favicon, so i was like why not use it 😄

Reply:
Oh :) 

> it was autogenerated by the tool i used to create favicon, so i was like why not use it

Could you add link for that somewere here (I think icons in index.twig would be the best) so it can be automated in the future?